### PR TITLE
Feature/wizard about page trng1022

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
* added about page design
* added new CreatorLayout class
* implemented proper link drawing

### CODE REVIEW

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs [hunk 1]
```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
 
         public override void Draw(Rect window)
         {
+            GUILayout.BeginArea(window);
+                GUILayout.Label("Hit Play to Preview", CreatorEditorStyles.Title);
+                GUILayout.Label("Have a look at How-Tos and an in-depth Webinar for further information.", CreatorEditorStyles.Paragraph);
+                GUILayout.Label("How-To's", CreatorEditorStyles.Header);
 
+                CreatorLayout.DrawLink("How to build your VR Training application", "https://developers.innoactive.de/documentation/creator/v2.2.0/articles/getting-started/designer.html");
```
Comment (Maxsiom) [inline Editor/UI/Wizard/FirstTime/AllAboutPage.cs:21]:
```suggestion
                CreatorLayout.DrawLink("How to build your VR Training application", "https://developers.innoactive.de/documentation/creator/latest/articles/getting-started/designer.html");
```

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs [hunk 2]
```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
...
+                CreatorLayout.DrawLink("How to extend the Creator using a training template", "https://developers.innoactive.de/documentation/creator/v2.2.0/articles/developer/01-introduction.html");
```
Comment (Maxsiom) [inline Editor/UI/Wizard/FirstTime/AllAboutPage.cs:22]:
```suggestion
                CreatorLayout.DrawLink("How to extend the Creator using a training template", "https://developers.innoactive.de/documentation/creator/latest/articles/developer/01-introduction.html");
```

**Comment by SimonTheSourcerer** [comment]
Unbenannt

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs [hunk 3]
```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
 
         public override void Draw(Rect window)
         {
+            GUILayout.BeginArea(window);
+                GUILayout.Label("Hit Play to Preview", CreatorEditorStyles.Title);
+                GUILayout.Label("Have a look at How-Tos and an in-depth Webinar for further information.", CreatorEditorStyles.Paragraph);
```
Comment (tomwim) [inline Editor/UI/Wizard/FirstTime/AllAboutPage.cs:18]:
```suggestion
                GUILayout.Label("Have a look at How-To's and an in-depth Webinar for further information.", CreatorEditorStyles.Paragraph);
```

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs [hunk 4]
```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
...
+                GUILayout.Label("How-To's", CreatorEditorStyles.Header);
 
+                CreatorLayout.DrawLink("How to build your VR Training application", "https://developers.innoactive.de/documentation/creator/latest/articles/getting-started/designer.html");
+                CreatorLayout.DrawLink("How to extend the Creator using a training template", "https://developers.innoactive.de/documentation/creator/latest/articles/developer/01-introduction.html");
+
+                GUILayout.Label("Need Help?", CreatorEditorStyles.Header);
+
+                CreatorLayout.DrawLink("In-depth webinar on how the Creator works", () => {});
```
Comment (tomwim) [inline Editor/UI/Wizard/FirstTime/AllAboutPage.cs:26]: We do not have a webinar yet, right? So maybe comment this so it isn't shown

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs [hunk 5]
```diff
@@ -12,7 +13,23 @@ public AllAboutPage() : base("Step 4: Help & Documentation")
...
+                CreatorLayout.DrawLink("Visit our developer community", "https://spectrum.chat/innoactive-creator");
+
+                GUILayout.Space(CreatorEditorStyles.Ident);
+
+                GUILayout.Label("Also, if you are facing any issues, don't hesitate to reach out to use for support", CreatorEditorStyles.Label);
```
Comment (tomwim) [inline Editor/UI/Wizard/FirstTime/AllAboutPage.cs:31]:
```suggestion
                GUILayout.Label("Also, if you are facing any issues, don't hesitate to reach out to us for support", CreatorEditorStyles.Label);
```

File: Editor/UI/Wizard/FirstTime/AllAboutPage.cs (hunk 4 above)
Comment (SimonTheSourcerer) [reply Editor/UI/Wizard/FirstTime/AllAboutPage.cs]: added the link :)

File: Editor/UI/CreatorLayout.cs [hunk 6]
```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Diagnostics;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Innoactive.Creator.Core.Editor.UI
+{
+    public static class CreatorLayout
```
Comment (SimonTheSourcerer) [inline Editor/UI/CreatorLayout.cs:9]: should this be public or better move it internal? (which will also disallow us from using it in another components)

Comment (tomwim) [reply Editor/UI/CreatorLayout.cs:9]: I dont mind being it public but then a few lines of docu would be nice ;)

Comment (Maxsiom) [reply Editor/UI/CreatorLayout.cs:9]: For me it's okay to be public. So everyone can adapt their layout.

Comment (Gusinuhe) [reply Editor/UI/CreatorLayout.cs:9]: If we use it in other places then this is an util and either the name or namespace feels odd.

Comment (SimonTheSourcerer) [reply Editor/UI/CreatorLayout.cs:9]: namespace is wrong anyway :D

Comment (SimonTheSourcerer) [reply Editor/UI/CreatorLayout.cs:9]: the name itself is mostly based on the same naming scheme unity uses:
GUILayout
EditorGUILayout
CreatorLayout

File: Editor/UI/CreatorGUILayout.cs [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+            }, intend);
+        }
+
+        /// <summary>
+        /// Draws a clickable link which looks like a hyperlink.
+        /// </summary>
+        /// <param name="text">Text to be displayed</param>
+        /// <param name="action">action done on click</param>
+        /// <param name="intend">Intend on the left</param>
+        public static void DrawLink(string text, Action action, int intend = CreatorEditorStyles.Indent)
```
Comment (tomwim) [inline Editor/UI/CreatorGUILayout.cs:42]: *indent
Sorry, missed it before :/

File: Editor/UI/CreatorGUILayout.cs [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+    /// </summary>
+    public static class CreatorGUILayout
+    {
+        /// <summary>
+        /// Draws a clickable link which opens a website.
+        /// </summary>
+        /// <param name="text">Text to be displayed</param>
+        /// <param name="url">url to be opened inside the browser</param>
+        /// <param name="intend">Intend on the left</param>
+        public static void DrawLink(string text, string url, int intend = CreatorEditorStyles.Indent)
```
Comment (tomwim) [inline Editor/UI/CreatorGUILayout.cs:21]: *indent
Sorry, missed it before :/

File: Editor/UI/CreatorGUILayout.cs (hunk 7 above)
Comment (Gusinuhe) [reply Editor/UI/CreatorGUILayout.cs:42]: is it **intend**ed? 😂

**Pull request merged**